Resources/Catalog/FileCatalog.py

```diff
@@ -216,13 +234,9 @@ def w_execute( self, *parms, **kws ):
       resDict = {'Failed':failed, 'Successful':successful}
       return S_OK( resDict )
     else:
-      if failedCatalogs:
-        result = S_ERROR( 'Failed to execute on some catalogs' )
-        resDict = {'Failed':failedCatalogs, 'Successful':successfulCatalogs}
-        result['Value'] = resDict
-        return result
-      else:
-        return masterResult
+      # FIXME: Return just master result here. This is temporary as more detailed
+      # per catalog result needs multiple fixes in various client calls
+      return masterResult
```

Comment:
out of curiosity, can you point me at some client in this case ? I couldn't find them